dot/peerset/peerset.go

```diff
@@ -357,7 +354,7 @@ func (ps *PeerSet) reportPeer(change ReputationChange, peers ...peer.ID) error {
 func (ps *PeerSet) allocSlots(setIdx int) error {
 	err := ps.updateTime()
```

Comment:
nit: `if err := ps.updateTime(); err != nil {` for consistancy

Reply:
I prefer the split one:
```
err := ps.updateTime()
if err != nil
```
changing the occurrences of `if err := ps.updateTime(); err != nil {`

Reply:
I agree with the split one :+1: 
Although I am an old [`ifshort`](https://github.com/esimonov/ifshort) user, so I have tried both teams :smile: